content/reference/user/dedicated_ips/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Dedicated IPs"
+description: "Dedicated IP addresses for your pools"
+lead: "Dedicated IP addresses for your pools"
+date: 2021-03-10T09:29:04-05:00
+lastmod: 2021-03-10T09:29:04-05:00
+weight: 10061
+---
+
+Dedicated IPs allows you to have control over the injectors IP addresses.
+
+{{< img src="dedicated-ips.png" alt="Dedicated IPs" caption="Dedicated IPs" >}}
+
+## Managing
+
+To access the dedicated IPs section, click on **Dedicated IPs** in the navigation bar.
+
+{{< alert tip >}}
+You can request dedicated IPs through [technical support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8/group/12/create/59).
+
+Please provide:
+- Organization SLUG
+- Desired number of dedicated IPs per region
+- Contact Email
+- GitHub username
+A sales person will contact you.
+{{< /alert >}}
+  
+Table present your available dedicated IPs address by pools.
+
+## Usage
+
+You can enable usage of dedicated IPs while [configuring simulation pools]({{< ref "../simulations#step-3-pools-configuration" >}}).
+
+When starting a run of a simulation configured to use dedicated IPs, 
+if there's enough of them by configured pools sizes, they will be reserved for the run duration.
+Otherwise, run won't be started.
```

Comment:
```suggestion
When starting a run of a simulation configured to use dedicated IPs, if you have enough dedicated IPs available to satisfy the configured pool sizes,
they will be reserved for the run duration. Otherwise, the run won't start.
```